src/screens/Settings/components/GeneralSettings/index.test.tsx

```diff
@@ -45,7 +45,7 @@ async function renderGeneralSettings(props: Partial<Props> = {})
     toggleDarkTrayIcon: () => {return;},
     toggleTray: () => {return;}
   };
-  const returnvalue = await render(<GeneralSettings {...{...defaultprops, ...props}} />);
+  const returnvalue = render(<GeneralSettings {...{...defaultprops, ...props}} />);
```

Comment:
Why?

Reply:
Same question here.

Reply:
Simple, because render is not an async function.
On your VScode cant you see like 3 dots on the await word and putting the mouse over it says: the await here does any difference, or something like this?